lib/usd/translators/jointWriteUtils.h

```diff
@@ -0,0 +1,172 @@
+//
+// Copyright 2016 Pixar
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// Modifications copyright (C) 2020 Autodesk
+//
+
+#ifndef PXRUSDMAYA_JOINT_WRITE_UTILS_H
+#define PXRUSDMAYA_JOINT_WRITE_UTILS_H
+
+#include <mayaUsd/base/api.h>
+
+#include <maya/MDagPath.h>
+#include <maya/MObject.h>
+#include <maya/MFnMesh.h>
+#include <maya/MString.h>
+
+#include <pxr/pxr.h>
+#include <pxr/base/gf/vec3f.h>
+#include <pxr/base/tf/staticTokens.h>
+#include <pxr/base/tf/token.h>
+#include <pxr/base/vt/array.h>
+#include <pxr/usd/usd/attribute.h>
+#include <pxr/usd/usdGeom/mesh.h>
+#include <pxr/usd/usdUtils/sparseValueWriter.h>
+#include <pxr/usd/usdSkel/bindingAPI.h>
+#include <pxr/usd/usdSkel/root.h>
+#include <pxr/usd/usdSkel/utils.h>
+#include <pxr/usd/usdSkel/skeleton.h>
+
+PXR_NAMESPACE_OPEN_SCOPE
+
+// Utilities for dealing with writing out joint and skin data.
+namespace UsdMayaJointUtil
```

Comment:
The namespace should ideally align with the file name, so `UsdMayaJointWriteUtils` in this case.

Reply:
Yes, I agree with your point but currently jointWriteUtils has a mixed of read and write functions and for the sake of time I didn't want add an extra work to separate them into to read/write namespaces. This can be broken down to jointWriteUtils [*.h,*.cpp] and jointReadUtils[*.h,*.cpp]
As I mentioned in the PR description, there is room for improvements here but this is something for another PR.